teach/views.py

```diff
@@ -140,9 +147,12 @@ def json_response(res, data):
 
 def info_for_user(res, user):
     token, created = Token.objects.get_or_create(user=user)
+    credly = get_credly_access_token(user.id)
     body = {
+        'username': user.username,
+        'email': user.email,
         'token': token.key,
-        'username': user.username
+        'credly': credly
```

Comment:
I'm pretty sure the changes in this file can be undone, the teach view has no business with credly or badging.